src/installer.ts

```diff
@@ -8,7 +8,7 @@ import * as tc from "@actions/tool-cache";
 import { Octokit } from "octokit";
 import * as semver from "semver";
 
-import { getAtmosBinaryName, getAtmosWrappedBinaryName } from "./atmos-bin";
+import { getAtmosBinaryName } from "./atmos-bin";

```

Comment:
I think we don't need this complicated logic anymore because we are removing the wrapper? We just need to see if it's windows and add `.exe`, otherwise it's atmos?